modules/global/src/com/haulmont/cuba/core/global/View.java

```diff
@@ -49,16 +49,21 @@
      * Parameters object to be used in constructors.
      */
     public static class ViewParams {
-        protected View src;
+        protected List<View> src;
         protected Class<? extends Entity> entityClass;
         protected String name;
         protected boolean includeSystemProperties;
 
         public ViewParams src(View src) {
-            this.src = src;
+            this.src = new ArrayList<>();

```

Comment:
Why do we need new ArrayList() here. Is it not enough to use Collections.emptyList()?